src/lib/mdns/minimal/Server.cpp

```diff
@@ -113,6 +113,12 @@ const char * AddressTypeStr(chip::Inet::IPAddressType addressType)
     }
 }
 
+static void ShutdownEndpoint(mdns::Minimal::ServerBase::EndpointInfo & aEndpoint)
```

Comment:
static not required in anonymous namespace

Reply:
Good catch, fixed.